src/main/kotlin/mate/academy/Main.kt

```diff
@@ -1,5 +1,21 @@
 package mate.academy
 
+import mate.academy.model.Figure
+import mate.academy.service.ColorSupplier
+import mate.academy.service.FigureSupplier
+
 fun main() {
-    // draw all figures here
+    val colorSupplier = ColorSupplier()
+    val figureSupplier = FigureSupplier(colorSupplier)
+    val numberOfFigures = 6
+    val numberOfRandomFigures = numberOfFigures / 2
+    val figures = Array<Figure>(numberOfFigures) { index ->
+        if (index < numberOfRandomFigures) {
+            figureSupplier.getRandomFigure()
+        } else {
+            figureSupplier.getDefaultFigure()
+        }
+    }
+
+    figures.forEach { it.draw() }
```

Comment:
You can get random figure and draw it immediately in cycle avoiding creation of array